cli/uninstall/uninstall_test.go

```diff
@@ -143,3 +143,35 @@ func TestSearchLatestVersion(t *testing.T) {
 		})
 	}
 }
+
+func TestGetAllVersionFormats(t *testing.T) {
+	type testCase struct {
+		name             string
+		ttVersion        string
+		expectedVersions []string
+		expectedError    error
+	}
+
+	cases := []testCase{
+		{
+			name:             "without prefix",
+			ttVersion:        "1.2.3",
+			expectedVersions: []string{"v1.2.3", "1.2.3"},
```

Comment:
There should be reverse priority here because we want to prefer to delete a specified version:
```suggestion
			expectedVersions: []string{"1.2.3", "v1.2.3"},
```

Reply:
Changed to this behaviour.